src/types.ts

```diff
@@ -317,6 +325,10 @@ export default {
    *   requiredEnumAsConstArray: types.enum(SampleArray, { required: true }),
    *   // type: 'foo' | 'bar' | undefined
    *   optionalEnumAsConstArray: types.enum(SampleArray),
+   *   // type: 'foo' | 'bar'
+   *   requiredEnumAsStringLitteralArray: types.enum(['foo', 'bar'], { required: true }),
```

Comment:
Let's keep this code consistent and define this array above as a variable. Let's also rename the previous `SampleArray` to `SampleArrayOfConst`.